### PR TITLE
Upgrade agp

### PR DESCRIPTION
**Pull request by paulwang007**

### CODE REVIEW

File: quality/errorprone/android.gradle [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+    annotationProcessor 'com.uber.nullaway:nullaway:0.6.5'
+    testAnnotationProcessor 'com.uber.nullaway:nullaway:0.6.5'
 }
 
 tasks.withType(JavaCompile) {
-  options.errorprone.errorproneArgs << '-XepDisableWarningsInGeneratedCode'
-  options.errorprone.errorproneArgs << '-Xep:NullAway:ERROR' << '-XepOpt:NullAway:AnnotatedPackages=com.uber'
+    // Adding (-Xep:ParameterName:OFF) due to issue: https://github.com/google/error-prone/issues/780.
+    options.errorprone.errorproneArgs << '-XepDisableWarningsInGeneratedCode' << '-Xep:ParameterName:OFF'
+    options.errorprone.errorproneArgs << '-Xep:NullAway:ERROR' << '-XepOpt:NullAway:AnnotatedPackages=com.uber'
```
Comment (NemoOudeis) [inline quality/errorprone/android.gradle]: why all the indentation changes? can you retain the original indentation in order to not pollute the diff & history?

- [ ] fix indentation

Comment (paulwang007) [reply quality/errorprone/android.gradle]: fixed

**Comment by paulwang007** [comment]
> 2 more points:
> 
> * [ ]  you have a lot of commits, some are unnecessary like the merge commits (just rebase) and one has a slightly incorrect commit message ([7fc7136](https://github.com/rakutentech/android-buildconfig/commit/7fc7136a6d5f8876654382fa5333a934743c811d) -> "Upgrade gradlew to 5.2.1"). can you either cherry pick the actual changes onto current master (i.e. no merge commits) or just squash it all into 1 commit (because that would still be a self contained commit imo)
> * [ ]  **DO NOT CLOSE THIS PR AND THEN OPEN ANOTHER ONE!** Please just push (with `--force` if necessary) to your fork's branch and this pull request will be updated. By closing this PR and opening a new one all the previous conversation is lost, that's kinda using the tool to our disadvantage.

**Comment by paulwang007** [comment]
All comments are addressed.

**Comment by NemoOudeis** [comment]
you can just check the box 😉

**Comment by paulwang007** [comment]
> you can just check the box 😉

I don't know how to click it, it doesn't seem to let me do it. Even reply to this message can't be done for some reason.

screen shot 2019-03-01 at 09 25 35

**Pull request merged**